docs/operators/csvParserOperator.md

```diff
@@ -0,0 +1,99 @@
+         "maxLength":"10"
+      }
+    },
+    {
+      "name":"bidPrice",
+      "type":"Double",
+      "constraints":{
+         "required":"true",
+         "minValue":"0.1",
+         "maxValue":"3.2"
+      }
+    },
+    {
+      "name":"startDate",
+      "type":"Date",
+      "constraints":{
+         "format":"yyyy-MM-dd HH:mm:ss",
+         "locale":"IN"
+      }
+    }
+  ]
+}
+```
+
+| **Property** | **Description** | **Type** | **Mandatory** | **Default Value** |
+| -------- | ----------- | ---- | ------------------ | ------------- |
+| *schema* | [Schema](https://github.com/apache/apex-malhar/blob/master/contrib/src/main/java/com/datatorrent/contrib/parser/DelimitedSchema.java)  describing delimited data | String | YES | N/A |
+ 
+## Platform Attributes that influences operator behavior
+
+| **Attribute** | **Description** | **Type** | **Mandatory** |
+| -------- | ----------- | ---- | ------------------ |
+| *out.TUPLE_CLASS* | TUPLE_CLASS attribute on output port which tells operator the class of POJO which need to be emitted | Class| Yes |
+ 
+## <a name="constraints"></a>Schema Constraints 
+|**Type**| **Constraints** | **Description** |
+| -------- | -------- | -------- |
+| *required* | True or False | If mentioned, indicates that the data type constraints that follow are required. |
+| *equals* | True or False | If mentioned, indicates that the data string or value declared in the data type constraints must be an exact match with the specified value. `Note: This constraints is not applicable for data type boolean and date` |
+| *String* | Length | The string must be of the length that is specified. |
+| *String* | minLength | The string is at least the length specified as minLength value. |
+| *String* | maxLength | The string can be at the most the length specified as maxLength value. |
+| *String* | pattern | The string must match the specified regular expression. |
+| *Long* | maxValue | The numeric can be at the most the value specified as maxValue. |
+| *Long* | minValue | The numeric is at least the value specified as minValue. |
+| *Double* | maxValue | The numeric can be at the most the value specified as maxValue. |
+| *Double* | minValue | The numeric is at least the value specified as minValue. |
+| *Integer* | maxValue | The numeric can be at the most the value specified as maxValue. |
+| *Integer* | minValue | The numeric is at least the value specified as minValue. |
+| *Date* | format | A simple date format as specified in the SimpleDateFormat class: http://docs.oracle.com/javase/8/docs/api/java/text/SimpleDateFormat.html?is-external=true |
+| *Boolean* | trueValue | String for which boolean value is true. The default values are: true, 1, y, and t. `Note: If you specify trueValue, you must also specify falseValue.` |
+| *Boolean* | falseValue | String for which boolean value is false. The default values are: false, 0, n, and f. `Note: If you specify falseValue, you must also specify trueValue.` |
+
```

Comment:
Add Float and Character as well. Float supports same constraints as Double. Character supports required and equals